packages/terra-form-fieldset/src/Fieldset.jsx

```diff
@@ -78,7 +78,7 @@ const Fieldset = ({
       {legend && <legend id={legendId} {...legendAttrs} className={legendClasses}>{legend}</legend>}
       {help && <small id={helpId} className={cx('help-text')} tabIndex="-1">{help}</small>}
       <div className={cx('fieldset-children')}>
-        {addAriaHelpers(children)}
+        {help || legend ? addAriaHelpers(children) : children}
```

Comment:
```suggestion
        {(help || legend) && children ? addAriaHelpers(children) : children}
```
this will protect the component from calling clone function on empty elements